packit/api.py

```diff
@@ -202,7 +202,9 @@ def sync_release(
         current_up_branch = self.up.active_branch
         try:
 
-            if not use_local_content:
+            # we want to check out the tag only when local_content is not set
+            # and it's an actualy upstream repo and not source-git
+            if not use_local_content and not upstream_ref:
```

Comment:
What about printing some `info` message to the user that we are doing this?

Reply:
OK, I now see that you mention that in the help. That's maybe enough.

Reply:
yeah, I can do that but the point here is that we are *not* doing the checkout :D